fplus-http-server/src/router/application.rs

```diff
@@ -523,8 +560,10 @@ pub async fn submit_kyc(info: web::Json<SubmitKYCInfo>) -> impl Responder {
         Err(e) => return HttpResponse::BadRequest().body(e.to_string()),
     };
     match ldn_application.submit_kyc(&info.into_inner()).await {
-        Ok(()) => HttpResponse::Ok()
-            .body(serde_json::to_string_pretty("Address verified with score").unwrap()),
+        Ok(_) => match serde_json::to_string_pretty("Address verified with score") {
```

Comment:
to_string_pretty won't fail if passed a static string. It's a good candidate for .expect.